languages/woocommerce-myparcelbe-fr_BE.po

```diff
@@ -198,6 +198,11 @@ msgstr "Défaut"
 msgid "Default export settings"
 msgstr "Paramètres d'exportation standards"
 
+msgid "product_option_hs_code_helpt_text_%stariff_website%s"
```

Comment:
Ik denk niet dat dit een goed idee is, kun je die variabelen beperken tot de daadwerkelijke inhoud van de vertaling?